### ISSUE TITLE
Crash on new version

### ISSUE THREAD

**Issue by m0ngr31** [mzomparelli/zVirtualDesktop#85]
```
Application: zVirtualDesktop.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.NullReferenceException
   at zVirtualDesktop.Program.StopMouseHook()
   at zVirtualDesktop.HookHandlers.HookManager_KeyDown(System.Object, System.Windows.Forms.KeyEventArgs)
   at zVirtualDesktop.HookManager.KeyboardHookProc(Int32, Int32, IntPtr)
```

**Comment by mzomparelli** [comment]
Thanks! I'm working on this now. You can test for me :)

**Comment by mzomparelli** [comment]
A crash can't happen there anymore. How fast are pressing modifier keys to make that happen lol

https://zomp.co/files.aspx?id=zvd

**Comment by mzomparelli** [comment]
Fixed in version 1.0.63
